test/spec/BpmnPropertiesPanel.spec.js

```diff
@@ -135,6 +177,32 @@ describe('<BpmnPropertiesPanel>', function() {
     });
 
 
+    it('should update on selection changed - multiple', async function() {
```

Comment:
Do we consider this as a breaking change (`propertiesPanel.updated` can now also contain multiple elements)? 🤔